test/machine_test.go

```diff
@@ -211,3 +269,14 @@ func updateSingleTest(key string, existing map[string]expected, val expected) ma
 	newExisting[key] = val
 	return newExisting
 }
+
+func sortedResources(m map[string]expected) []string {
+	keys := make([]string, len(m))
+	i := 0
+	for k := range m {
+		keys[i] = k
+		i++
+	}
+	sort.Strings(keys)
+	return keys
+}
```

Comment:
this is dead code now, lets delete it

Reply:
this is in use at line 153 https://github.com/tinkerbell/pbnj/blob/ac26caea37e6b20ae85ddae7f2e3099efdf5b7c6/test/machine_test.go#L153

Reply:
oh right I thought is was `doesNotCoatainTrue`